bash-preexec.sh

```diff
@@ -199,14 +200,18 @@ __bp_preexec_invoke_exec() {
         if type -t "$preexec_function" 1>/dev/null; then
             __bp_set_ret_value $__bp_last_ret_value
             $preexec_function "$this_command"
-            preexec_ret_value="$?"
+            preexec_function_ret_value="$?"
+            if (( $preexec_function_ret_value != 0 )); then
```

Comment:
The round brackets are actually a different construct (arithmetic expansions - http://tldp.org/LDP/abs/html/arithexp.html#ARITHEXPREF) that seem pretty reasonable here, since the variable used is guaranteed to be an integer. Doing it the other way would be a string comparison, which works, but I think this approach is preferable. Happy to be overruled on this, as well.